tests/playwright/src/specs/extension-installation.spec.ts

```diff
@@ -83,19 +91,40 @@ describe.each([
     await navBar.openExtensions();
   });
 
-  test('Install extension through Extensions Catalog', async () => {
-    const extensionsPage = new ExtensionsPage(page);
-
-    await extensionsPage.openCatalogTab();
-    const extensionCatalog = new ExtensionCatalogCardPage(page, extensionType);
-    await playExpect(extensionCatalog.parent).toBeVisible();
-
-    await playExpect.poll(async () => await extensionCatalog.isInstalled()).toBeFalsy();
-    await extensionCatalog.install(180000);
-
-    await extensionsPage.openInstalledTab();
-    await playExpect.poll(async () => await extensionsPage.extensionIsInstalled(extensionLabel)).toBeTruthy();
-  }, 200000);
+  test.runIf(extensionType.indexOf(OPENSHIFT_LOCAL) >= 0 || extensionType.indexOf(OPENSHIFT_SANDBOX) >= 0)(
```

Comment:
I'm not sure I understand what the point of this is

Reply:
Different extension, different check. Given we have one test file, not all tests can be done for all extensions. Ie. openshift local and sandbox have provider cards on dashboard page, while openshift checker does not.

Reply:
No, that I understand, what I dont get is why didn't you just check `extensionType === OPENSHIFT_LOCAL`  or whatever, I dont understand this use of indexOf >= 0

Reply:
this is the whatever. Seems like development of the condition, first it was just searching for substring, later it became constant. so I just kept it there.

Reply:
triple equals sign placed.